src/components/buttons/AddEmployeeButton/AddEmployeeButton.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './AddEmployeeButton.css';
+import { IoIosAddCircleOutline } from 'react-icons/io'
+
+const AddEmployeeButton = ({
+    text
+}) => {
+
+    function handleClickPopup() {
```

Comment:
I am not quite sure I know what this one is responsible for. Can you elaborate? 🍏